rosdep/base.yaml

```diff
@@ -4857,21 +4855,13 @@ wxwidgets:
   opensuse: [wxGTK-devel]
   rhel: [wxGTK-devel]
   ubuntu:
+    '*': [libwxgtk3.0-dev]
     lucid: [libwxgtk2.8-dev]
-    maverick: [libwxgtk2.8-dev]
-    natty: [libwxgtk2.8-dev]
-    oneiric: [libwxgtk2.8-dev]
     precise: [libwxgtk2.8-dev]
-    quantal: [libwxgtk2.8-dev]
-    raring: [libwxgtk2.8-dev]
-    saucy: [libwxgtk2.8-dev]
```

Comment:
What is the rational for removing these entries? It break users on that platform for no real benefit (beyond saving a few lines).

Reply:
Their EOLs were May 2014, Jan 2014, and Jul 2014, respectively; I just thought they were way outdated. But it's true that it doesn't imply nobody is using them in production.

I'll add one further commit just adding what I originally needed: support for artful and bionic, leaving the rest untouched, so you can merge with a squash and avoid all this noise. Thanks for the feedback.